web/src/components/molecules/Schema/FieldModal/FieldDefaultInputs/TextField/index.tsx

```diff
@@ -2,14 +2,19 @@ import React from "react";
 
 import Form from "@reearth-cms/components/atoms/Form";
 import Input from "@reearth-cms/components/atoms/Input";
+import MultiValueField from "@reearth-cms/components/molecules/Common/MultiValueField";
 import { useT } from "@reearth-cms/i18n";
 
-const TextField: React.FC = () => {
+type Props = {
+  multiple?: boolean;
+};
+
+const TextField: React.FC<Props> = ({ multiple }) => {
   const t = useT();
 
   return (
     <Form.Item name="defaultValue" label={t("Set default value")}>
-      <Input />
+      {multiple ? <MultiValueField FieldInput={Input} /> : <Input />}
```

Comment:
Again, seems weird to be passing the atom component to MultiValueField.
Passing a type?: string prop is maybe enough?